src/scss/06_components/paragraphs/_content-cards.scss

```diff
@@ -126,29 +122,31 @@ $-card-spacing: $spacing-and-half;
 
 .component--content-cards-grey {
   background: $color-silver-light;
+  display: grid;
+  grid-template-columns: [full-width-start] $spacing [content-start] 1fr [content-end] $spacing [full-width-end] ;
+  grid-template-rows: [header-start] auto [header-end] auto;
+  padding-top: 0;
+
+  @include breakpoint($breakpoint-m) {
+    grid-template-columns: [full-width-start] $spacing-double [content-start] 1fr [content-end] $spacing-double [full-width-end] ;
```

Comment:
I'd consider using a custom property (css variable) for the gap, so that only it changes and we do not need to define the full `grid-template-columns` twice. Maybe set it without the mediaquery as 
```
grid-template-columns: [full-width-start] var(--component-gap, #{$spacing}) [content-start] 1fr [content-end] var(--component-gap, #{$spacing}) [full-width-end];
```
 and then in the mediaquery just set `--component-gap: #{$spacing-double};`. This way it's easier to see what we're doing at the breakpoint and to adjust the template-columns if needed.